templates/error.html

```diff
@@ -0,0 +1,418 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Sivustolla on häiriö</title>
+    <style>
+      body {
+        font-family: Arial, sans-serif;
+        background-color: #f0f0f0;
+        text-align: center;
+        margin: 0;
+        padding: 0;
+      }
+
+      header {
+        display: flex;
+        height: 88px;
+        flex-direction: column;
+        justify-content: center;
+        align-items: flex-start;
+        align-self: stretch;
+        background: #fff;
+        width: 100%;
+        border-bottom: 1px solid #ccc;
+
+        .site-title {
+          display: flex;
+          padding: 16px 0px;
+          align-items: center;
+          font-size: 24px;
+          font-style: normal;
+          font-weight: 500;
+          line-height: 24px;
+          letter-spacing: -0.048px;
+        }
+
+        .logo-wrapper {
+          display: flex;
+          justify-content: center;
+          align-items: center;
+        }
+
+        .hel-icon {
+          margin-left: 24px;
+          margin-right: 16px;
+        }
+
+        .hel-icon-svg {
+          height: 56px;
+          width: 122px;
+        }
+
+        .city-name,
+        a {
+          color: #1a1a1a;
+          text-decoration-line: none;
+        }
+      }
+
+      @media only screen and (max-width: 768px) {
+        header {
+          height: 64px;
+
+          .hel-icon {
+            margin-left: 16px;
+            height: 32px;
+            width: 70px;
+          }
+
+          .hel-icon-svg {
+            height: 32px;
+            width: 70px;
+          }
+
+          .city-name {
+            display: none;
+          }
+        }
+      }
+
+      .main-wrapper {
+        color: #1a1a1a;
```

Comment:
Text color is the same everywhere than footer, so may define it once in body and once in footer? #1a1a1a is repeated quite often.

Reply:
Should be better now